modules/@apostrophecms/form-group-widget/views/widget.html

```diff
@@ -0,0 +1,13 @@
+{% set prependIfPrefix = apos.modules['@apostrophecms/form'].prependIfPrefix %}
+
+<div data-apos-form-groups class="apos-form-groups {{ prependIfPrefix('__groups') }}">
+  {% for group in data.widget.groups %}
```

Comment:
Nice. I guess from the POV of the code that handles form submissions it just hoovers up the nested fields like everything else?

Reply:
Yep